components/tag-list/test/tag-list.visual-diff.js

```diff
@@ -21,28 +21,28 @@ describe('d2l-tag-list', () => {
 
 	it('is correct at 1400px page width', async function() {
 		const rect = await visualDiff.getRect(page, '#default');
-		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { captureBeyondViewport: false, clip: rect });
```

Comment:
Trying to address some flake here:
https://visual-diff.d2l.dev/screenshots/BrightspaceUI/core/tag-list/2022-05-02-20.34.45.71-d27e74a00cf834822f1857e5a2ecf216b200f8ca-report.html